#
# On POWER this patch also fixes test-ildoubl and test-ldouble failures where tan
# rounded toward zero had acceptable 1 ULP error. Upstream is using 3 ULP, but
# we prefer to keep the bound tighter unless we have a reason not to.
#
# This is the ppc64le version which is required becuase it applies *after* another
# ppc64le patch that touches the same ULPs file. See glibc-power-libm-test-ulps.patch
# for the ppc64/ppc version.
#
diff -urN glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps
--- glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps	2014-07-25 22:07:06.280020855 -0400
+++ glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps	2014-07-25 22:26:54.650021033 -0400
@@ -2644,6 +2644,9 @@
 Test "tan_towardzero (2)":
 ildouble: 1
 ldouble: 1
+Test "tan_towardzero (2) == -2.1850398632615189916433061023136825434320":
+ildouble: 1
+ldouble: 1
 Test "tan_towardzero (3) == -0.1425465430742778052956354105339134932261":
 float: 1
 ifloat: 1
